java/android/nnstreamer/src/main/jni/Android.mk

```diff
@@ -41,7 +41,17 @@ NNSTREAMER_API_OPTION := all
 ENABLE_TENSOR_QUERY := true
 
 # tensorflow-lite (nnstreamer tf-lite subplugin)
+ifdef TFLITE_ROOT_ANDROID
+ifneq ($(filter $(TARGET_ARCH_ABI), armeabi-v7a arm64-v8a),)
```

Comment:
Is this compatible with recent Android devices? E.g., Galaxy S24 has armv9 CPU cores. I'm not sure if such devices use binaries of arm64-v8a or v9a.